__tests__/app.test.js

```diff
@@ -1,217 +1,301 @@
+      username: "butter_bridge",
+    };
+    return request(app)
+      .post("/api/articles/4/comments")
+      .send(newComment1)
+      .expect(201)
+      .then(({ body }) => {
+        const { comment } = body;
+        expect(comment).toMatchObject({
+          comment_id: 19,
+          body: "An insightful article shedding light on the latest breakthrough in renewable energy technology!",
+          article_id: 4,
+          author: "butter_bridge",
+          votes: 0,
+          created_at: expect.any(String),
+        });
+        const newComment2 = {
+          body: "This news undoubtedly opens up new possibilities and promises exciting advancements in the future.",
+          username: "icellusedkars",
+        };
         return request(app)
-        .get('/api/notARoute')
-        .expect(404)
-        .then(({body}) => {
-            expect(body.msg).toBe('Not found')
-        })
-    })
-})
+          .post("/api/articles/6/comments")
+          .send(newComment2)
+          .expect(201)
+          .then(({ body }) => {
+            const { comment } = body;
+            expect(comment).toMatchObject({
+              comment_id: 20,
+              body: "This news undoubtedly opens up new possibilities and promises exciting advancements in the future.",
+              article_id: 6,
+              author: "icellusedkars",
+              votes: 0,
+              created_at: expect.any(String),
+            });
+          });
+      });
+  });
+  test("status:400, when passed an empty body should return error msg Bad request", () => {
+    const newComment3 = {
+      body: "",
+      username: "icellusedkars",
+    };
+    return request(app)
+      .post("/api/articles/6/comments")
+      .send(newComment3)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad request");
+      });
+  });
+  test("status:404, when passed an invalid username should return error msg Not found", () => {
+    const newComment4 = {
+      body: "Let's continue to engage with these important narratives and work together towards a more informed and empathetic society.",
+      username: "bobthebuilder",
+    };
+    return request(app)
+      .post("/api/articles/8/comments")
+      .send(newComment4)
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Not found");
+      });
+  });
+  test("status:404, should return an error msg with Not found when the ID does not exist", () => {
+    const newComment5 = {
+        body: "This article provides valuable insights into the latest developments, offering a well-rounded perspective on the events shaping our present.",
+        username: "icellusedkars",
+      };
+    return request(app)
+      .post("/api/articles/999999/comments")
+      .send(newComment5)
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Not found");
```

Comment:
Really great work with this test suite! Would be good to also see a test for an invalid ID as well please.
It may also be worth considering a test for making sure that we ignore extra, unnecessary properties on the input body too.